modules/xfem/test/tests/switching_material/two_cuts_moving.i

```diff
@@ -0,0 +1,135 @@
+[Problem]
+  solve = false
+[]
+
+[XFEM]
+  qrule = volfrac
+  output_cut_plane = true
+[]
+
+[UserObjects]
+  [cut1]
+    type = LevelSetCutUserObject
+    level_set_var = phi1
+    negative_id = 1
+    positive_id = 33
+    execute_on = NONE
+  []
+  [cut2]
+    type = LevelSetCutUserObject
+    level_set_var = phi2
+    negative_id = 5
+    positive_id = 16
+    execute_on = NONE
+  []
+  [combo]
+    type = ComboCutUserObject
+    geometric_cut_userobjects = 'cut1 cut2'
+    cut_subdomain_combinations = '1 5 1 16 33 5 33 16'
```

Comment:
We should probably break all of these up into separate lines for clarity (in this and the other input files). I'm guessing that hit format doesn't play nicely with that, though.

Reply:
This is now taking vector of vector of IDs per your suggestion. Input files are also updated.